pkg/ExternalPackages/versions.props

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
+  <ItemGroup>
+    <ExternalPackage Include="System.IO.Compression">
+      <Version>4.1.0-rc3-24117-00</Version>
```

Comment:
I mentioned that in my hand-off to @jhendrixMSFT , but I'll file an issue to make sure it doesn't get lost.